src/solvers/refinement/string_constraint_generator.cpp

```diff
@@ -0,0 +1,3349 @@
+/*******************************************************************\
```

Comment:
separated in 10 files https://github.com/diffblue/cbmc/pull/374/commits/92b8e0935d2e327e1d8af9e19768466d4a90bc01